QgisModelBaker/gui/panel/pg_config_panel.py

```diff
@@ -324,7 +353,7 @@ def _pg_service_combo_box_changed(self):
             service_config = pgserviceparser.service_config(service)
 
             # QGIS cannot handle manually set hosts with service
-            # So it needs to has a host defined in service conf or it takes localhost
+            # So it needs to have a host defined in service conf or it takes localhost
```

Comment:
Ah haven't known. You know why QGIS cannot handle it? Anyway, I guess in 99% of service conf use cases you have a host configured there :-)

Reply:
I'm a bit surprised as well. Just fixed the typo, but the note is a bit of a limitation for QGIS. BTW, I didn't actually try it.